src/main/java/Exception/BlitzException.java

```diff
@@ -0,0 +1,16 @@
+package exception;
+
+/**
+ * Represents a generic exception in the Blitz application.
+ */
+public class BlitzException extends Exception {
```

Comment:
All of the exceptions under BlitzException feel like RuntimeExceptions rather than checked exceptions.

Consider extending RuntimeException instead?